### ISSUE TITLE
Usage with vs2017 on new CRM

### ISSUE THREAD

**Issue by jarmoosh** [scottdurow/SparkleXrm#223]
Hello!

When I try to load SparkleXRM into VS2017 I receive notification and the solution won't build. Is it compatible with VS2017 and CRM 2016/ 365?

Best regards

**Comment by scottdurow** [comment]
Yes - I currently use VS2017 and Dynamics 365 V9 as it's development environment.
Can you try using the sparklexrm-dev branch?

**Comment by jarmoosh** [comment]
![obraz](https://user-images.githubusercontent.com/23243192/38935259-34ba6330-431e-11e8-9d17-50d0ba2c2c4d.png)
I get this kind of error

**Comment by lsuhadolnik** [comment]
Try installing Developer toolkit 365. You will have to modify the .vsix you download to allow installing to VS17

Download this:
https://marketplace.visualstudio.com/items?itemName=DynamicsCRMPG.MicrosoftDynamicsCRMDeveloperToolkit

Change the downloaded file extension from VSIX to ZIP, 
- extract it, 
- open .manifest file, 
- change Version="[11.0, 14.00]" to Version="[11.0, 15.0]"
- zip all files again and 
- change the extension to VSIX

The process should pass then.

Follow this guide: 
https://dynamics365northernireland.com/2017/03/07/install-dynamics-365-developer-toolkit-with-visual-studio-2017/
